recipe/patches/0001-Redis-deps-now-build-but-do-not-link.patch

```diff
@@ -1,27 +1,23 @@
-From 5872db3888c7fa3f4352464ae9520b7914bc39c3 Mon Sep 17 00:00:00 2001
-From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Fri, 3 Dec 2021 10:46:36 -0800
-Subject: [PATCH 1/5] Redis deps now build but do not link
+From eed3c457b01447e2c7f226006070a7e9eb454896 Mon Sep 17 00:00:00 2001
+From: Mohamed Sentissi <msentissi@anaconda.com>
```

Comment:
You should leave previous authors as co-authors at least.

Reply:
Sure ! I think this was overwritten because I had to regenerate the patches. Fixed.